paket.lock

```diff
@@ -1,17 +1,18 @@
 NUGET
   remote: https://www.nuget.org/api/v2
   specs:
-    FAKE (4.0.1)
-    FSharp.Compiler.Service (1.3.1.0)
-    FSharp.Formatting (2.9.10)
+    FAKE (4.3.1)
+    FSharp.Compiler.Service (1.4.0.1)
+    FSharp.Formatting (2.10.0)
```

Comment:
Can you roll this back, I'll update dependencies on master at some point but prefer not to conflate.